examples/pane-shell.yaml

```diff
@@ -0,0 +1,22 @@
+session_name: Pane shell example
+windows:
+  - window_name: first
+    window_shell: /usr/bin/python2
+    layout: even-vertical
+    suppress_history: false
+    options:
+      remain-on-exit: true
+    panes:
+      - shell: /usr/bin/python3
+        shell_command:
+          - print('This is python 3')
+      - shell: /usr/bin/vim -u none
+        shell_command:
+          - iAll panes have the `remain-on-exit` setting on.
```

Comment:
```suggestion
          - All panes have the `remain-on-exit` setting on.
```

Reply:
Wait. I'm wrong

Reply:
😆

Reply:
`i`... _Insert_

I've been in vim every day for 15+ years and this got passed me